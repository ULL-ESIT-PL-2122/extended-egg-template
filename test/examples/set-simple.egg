do(
    def(a, array(array(1,object(b: 5)),3)),
    set(a[0, 1, "b"], 3),
    print(a)
)