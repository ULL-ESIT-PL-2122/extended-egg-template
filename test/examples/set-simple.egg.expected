[[1,{"b":3}],3]
